src/kp2bw/bw_serve.py

```
@@ -0,0 +1,574 @@
+        # Terminate the process before raising so close() is clean.
+        if self._process is not None and self._process.poll() is None:
+            self._process.terminate()
+            try:
+                self._process.wait(timeout=5)
+            except subprocess.TimeoutExpired:
+                self._process.kill()
+                self._process.wait(timeout=2)
+        self.close()
+        raise BitwardenClientError(
+            f"bw serve did not become ready within {_SERVE_STARTUP_TIMEOUT_S}s"
+        )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)